app/controllers/sign_rebate_forms_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+class SignRebateFormsController < ApiController
+  jsonapi resource: SignRebateFormResource
+  strong_resource :rebate_form
+
+  def sign
+    token = params[:data][:token]
+
+    raise JsonapiCompliable::Errors::ValidationError unless token
+
+    decoded_token = decode_jwt(token)
+
+    rebate_form_id = decoded_token[0]['rebate_form_id']
```

Comment:
see previous comment

Reply:
okay again we aren't using JWT's anywhere else, but sure will future proof